_episodes/03-built-in.md

```diff
@@ -178,13 +178,14 @@ result of print is None
 ~~~
 {: .output}
 
-## Methods are functions for specific types
```

Comment:
comment on lesson order: You've moved the built-in lesson before talking about variable types. However, the built-in function and help lesson requires learners to be aware of the different types of variables. For example, the very first section of the lesson requires them to know that `string` is a type of variable and that it is different from an `int`. I don't think this should be taught before talking about variable types.